.github/workflows/test_federation.yml

```diff
@@ -60,12 +52,9 @@ jobs:
       - name: Lint
         run: cd federation && yarn && yarn run lint
 
-  ##############################################################################
-  # JOB: UNIT TEST #############################################################
-  ##############################################################################
   unit_test:
     name: Unit Tests - Federation
-    if: needs.files-changed.outputs.docker == 'true' || needs.files-changed.outputs.federation == 'true'
```

Comment:
missing dependency on database and mariadb

Reply:
Checking for file changes on database and mariadb Docker files will be handle in a follow up PR.
This PR is about removing unnecessary checks for changes on docker-compose files in build jobs.